### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by TheoLiu31** [elnino9ykl/DS-PASS#2]
Hi,
your swaft model works well with my fisheye test image, can you share your training code so I can add my own dataset to your pretrained model?
Thanks

**Comment by elnino9ykl** [comment]
Hi,

thank you for your comment.
I just uploaded the training code.
Please note that I have not performed thorough cleanings of the code.

You could also refer to 

https://github.com/elnino9ykl/PASS/tree/master/train

and

https://github.com/AHupuJR/RFNet

**Comment by TheoLiu31** [comment]
> Hi,
> 
> thank you for your comment.
> I just uploaded the training code.
> Please note that I have not performed thorough cleanings of the code.
> 
> You could also refer to
> 
> https://github.com/elnino9ykl/PASS/tree/master/train
> 
> and
> 
> https://github.com/AHupuJR/RFNet

Hi, 
thanks for your quick answer.
I notice that your class name/id does not seem exactly to cityscapes . 
Could you share me how do you generate annotations images and your dataset? 
Do you use cityscapesscripts to generate your dataset?
Thanks

**Comment by elnino9ykl** [comment]
We use Mapillary Vistas for training to take advantage of its diverse images.
We use the classes as described in our paper (Table I) https://arxiv.org/pdf/1909.07721.pdf
The data are structured the same as that of Cityscapes.
For cityscapes training, I think you only have to change the number of classes.
